tasks/create_nat_gateway.yml

```diff
@@ -58,9 +65,49 @@
     - natgw.eip_name is defined
 
 
+# Idempotence Issue
```

Comment:
IMHO, this should be part of the README, in some sort of a "caveats" section.

Reply:
Issue has now been resolved by this alternative approach.